include/common/aarch32/el3_common_macros.S

```diff
@@ -195,6 +195,18 @@
 	cps	#MODE32_mon
 	isb
 
+#if defined(IMAGE_BL32) && SP_MIN_WITH_SECURE_FIQ
```

Comment:
nit: Would prefer to avoid SP_MIN build flags in el3_common macros. Perhaps an additional argument to this macro (_route_fiq?) and SP_MIN entry point can encode this argument based on SP_MIN_WITH_SECURE_FIQ flag ?

Please dont do the rework yet. Let me confirm that the suggestion is alright.